titleMapping.json

```diff
@@ -239,6 +239,106 @@
 		"TocTitle": "Contexts",
 		"PageTitle": "Contexts"
 	},
+	"az datalake": {
+		"TocTitle": "Data Lake",
+		"PageTitle": "Data Lake"
+	},
+	"az datalake analytics": {
+		"TocTitle": "Analytics",
+		"PageTitle": "Data Lake Analytics"
+	},
+	"az datalake analytics account": {
+		"TocTitle": "Accounts",
+		"PageTitle": "Data Lake Analytics accounts"
+	},
+	"az datalake analytics account blob": {
+		"TocTitle": "Blobs",
+		"PageTitle": "Data Lake Analytics accounts linked to Azure Storage"
+	},
+	"az datalake analytics account datalake-store": {
+		"TocTitle": "Data Lake Store",
+		"PageTitle": "Data Lake Analytics accounts linked to Data Lake Store accounts"
+	},
+	"az datalake analytics account firewall": {
+		"TocTitle": "Firewall rules",
+		"PageTitle": "Firewall rules for Data Lake Analytics accounts"
```

Comment:
Should be the same pattern as the others:
Data Lake Analytics account firewall rules